src/cmd/preview.go

```diff
@@ -60,13 +66,15 @@ func PrintServices(isTextOutput bool, samples int, queue <-chan opslevel_jq_pars
 	fmt.Println(string(prettyJSON))
 
 	if isTextOutput {
-		servicesCount := len(*services)
-		if samples < servicesCount {
-			if samples == 0 {
-				samples = servicesCount
-			}
-			fmt.Printf("\nShowing %v / %v resources\n", samples, servicesCount)
+		var (
+			servicesCount int    = len(*services)
+			midText       string = "a random sample of "
```

Comment:
why are we doing this?  this makes the print out below incredibly complex to read for no reason

Reply:
@rocktavious letting the user know a random sample is being printed is necessary because it is confusing to see output that varies each time.

When I first ran preview I added about 20 pods to my cluster and noticed that the output was slightly different each time. This is because of the GetSample() function showing a random sample of 5 entries by default. 

Letting the user know explicitly that they are being shown a random sample makes this much more obvious.

Reply:
If the goal is to let the end user know a sampling is taken then let us just print that out.  The complexity of this along with the obtuse variable name `midText` is a code smell to me that we are doing unnecessary complexity.  All of it could be mitigated with just a simple print out of "here is a sampled results list - it will not contain the full list - if you want the full list please set the sample count to <= 0"

Reply:
> All of it could be mitigated with just a simple print out of something to the effect of "here is a sampled results list - it will not contain the full list - if you want the full list please set the sample count to <= 0"

Good idea. I just pushed a commit to do exactly this and get rid of `midText`.